test/parallel/test-vm-codegen.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const { createContext, runInContext, runInNewContext } = require('vm');
+
+const WASM_BYTES = Buffer.from(
+  [0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00]);
+
+
+function expectsError(fn, type) {
+  try {
+    fn();
+    assert.fail('expected fn to error');
+  } catch (err) {
+    console.log(err);
```

Comment:
If the `console.log()` is an intended part of the test, can you add a comment. Otherwise, please remove it.